### ISSUE TITLE
Unhandled exception in RatScanner 2.6.0

### ISSUE THREAD

**Issue by XenSide** [RatScanner/RatScanner#80]
Name scanning the key "RB-BK key" and clicking on the wiki link causes the application to throw an exception and close itself. 

EDIT: The same also happens with the "RB-AO key", log for RB-AO not added, I can add it later if needed.

Log:

**Error**
Unhandled exception in RatScanner 2.6.0
```
System.ComponentModel.Win32Exception (2): The system cannot find the file specified.
   at System.Diagnostics.Process.StartWithShellExecuteEx(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at RatScanner.View.MainMenu.HyperlinkRequestNavigate(Object sender, RequestNavigateEventArgs e)
   at System.Windows.Navigation.RequestNavigateEventArgs.InvokeEventHandler(Delegate genericHandler, Object genericTarget)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.ContentElement.RaiseEvent(RoutedEventArgs e)
   at System.Windows.Documents.Hyperlink.NavigateToUri(IInputElement sourceElement, Uri targetUri, String targetWindow)
   at System.Windows
```
<details>
<summary>Log</summary>

```
[22:47:13.2855104] > [Info]  ----- RatScanner 2.6.0 -----
[22:47:13.2861172] > [Info]  Starting RatScanner...
[22:47:13.2861548] > [Info]  Loading config...
[22:47:13.2973099] > [Info]  Loading item data...
[22:47:13.3020635] > [Info]  Loading resource "RSItemDataLink"...
[22:47:13.9273690] > [Info]  Downloading file "https://api.ratscanner.com/v3/file/items.json"...
[22:47:19.5260945] > [Info]  Loading price data...
[22:47:20.3539623] > [Info]  Setting default item...
[22:47:20.3663937] > [Info]  Initializing name scan...
[22:47:20.3665498] > [Info]  Loading OCR data...
[22:47:20.4131521] > [Info]  Initializing icon scan...
[22:47:20.4137915] > [Info]  Loading static icons...
[22:47:20.4957322] > [Info]  Loaded 1197/1197 icons successfully.
[22:47:20.4965466] > [Info]  Loading static correlation data...
[22:47:20.5101510] > [Info]  Loading dynamic correlation data...
[22:47:20.5208774] > [Info]  Loading dynamic icons...
[22:47:20.6210405] > [Info]  Loaded 2176/2176 icons successfully.
[22:47:20.6233439] > [Info]  Initializing file watcher for icon cache...
[22:47:20.6247022] > [Info]  Prewarming name tool tip...
[22:47:20.6550521] > [Info]  Prewarming icon tool tip...
[22:47:20.6701410] > [Info]  Initializing hotkey manager...
[22:47:20.6714816] > [Info]  Checking for new updates...
[22:47:20.6716115] > [Info]  Loading resource "RSClientVersion"...
[22:47:21.0430218] > [Info]  Ready!
[22:47:21.0587543] > [Info]  Loading resource "RSPatreonLink"...
[22:47:21.4473404] > [Info]  Loading resource "RSDiscordLink"...
[22:47:21.7983796] > [Info]  Loading resource "RSGithubLink"...
[22:47:24.8696911] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8698784] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8699243] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8699519] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8699913] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8700273] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8700592] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8701125] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8701470] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8701727] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8702327] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8702724] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8702990] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8703249] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8703495] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8703750] > [Warning] Could not find item with uid: 5a0448bc86f774736f14efa8
[22:47:24.8703995] > [
```
</details>

**Comment by Blightbuster** [comment]
Hey, im unable to reproduce this issue.
Do you by chance not have a default browser set?

**Comment by XenSide** [comment]
Microsoft Edge is my default browser and it does work as one

**Comment by Blightbuster** [comment]
Fixed in v2.7.0. Thanks for reporting ❤️
